pages/common/pytest.md

```diff
@@ -0,0 +1,28 @@
+# pytest
+
+> Run Python tests.
+> More information: <https://docs.pytest.org/>.
+
+- Run tests from a specific file:
+
+`pytest {{path/to/test_file.py}}`
+
+- Run all tests with a name that matches a given substring:
+
+`pytest -k {{substring}}`
```

Comment:
```suggestion
- Run tests with names matching a specific [k]eyword expression:

`pytest -k {{expression}}`
```

Optional change: `all` can be assumed by default implicitly.
Optional change (no suggestions available based on man page): provide examples instead of generic `expression` placeholder delimited by `|` char.